<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonSzukaj">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Szukaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPolacz">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Połącz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRozlacz">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rozłącz</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>621</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stan wyjść cyfrowych</string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1023</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zadajniki AC wyjść</string>
    </property>
   </widget>
   <widget class="QDial" name="dial_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1023</number>
    </property>
   </widget>
   <widget class="QDial" name="dial_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>560</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1023</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DO1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DO2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DO3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DO4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DO6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DO7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DO8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DO5</string>
    </property>
   </widget>
   <widget class="QLabel" name="StanWejsc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="wykres1" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>351</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
